### ISSUE TITLE
[BUG] - Multiple carousels lead to an a11y problem

### ISSUE THREAD

**Issue by MarcelKnobelsdorf** [porsche-design-system/porsche-design-system#3557]
### JS Framework

React

### Porsche Design System version

3.19.0

### Reproduction Example

https://stackblitz.com/run?file=App.tsx

### Steps to reproduce the behavior

https://designsystem.porsche.com/v3/components/carousel/examples

1. Go to https://stackblitz.com/run?file=App.tsx
2. run axe-core with best practices
3. Landmarks should have a unique role or role/label/title (i.e. accessible name) combination 

### Expected behavior

no a11y issues:
Enable to override aria-roledescription="carousel" to make it unique

**Comment by o-hook** [comment]
Hi @MarcelKnobelsdorf,
thanks for reporting. Do you refer to the implementation on our Storefront? If so, the A11y issue regarding unique landmarks is an implementation thing on our Storefront, because we are using always the same "heading" content for better and cleaner documentation. This heading text is then referred by the `aria-labelledby` attribute on the `role="region"` container.

Can you verify this? Thanks!

**Comment by MarcelKnobelsdorf** [comment]
Hey Oli, you're right. We also have/had the error in the store, but it was due to our structure. 

In the store we want to have the heading at the same line as the controls, to make this possible I have to remove the heading or rather I have now hidden it so that the text is still correct.
